plugins/inputs/intel_powerstat/sample.conf

```diff
@@ -25,3 +25,8 @@
   ##   "cpu_busy_frequency"
   ## ATTENTION: cpu_busy_cycles is DEPRECATED - use cpu_c0_state_residency
   # cpu_metrics = []
+
+  # The user can select from which cores he wants to collect metrics.
+  # The default is empty, which reports every core in the system.
+  # The user can set the cores as the example below.
+  # cores = ["0,3", "10", "20-30"]
```

Comment:
The commented value should represent the default value. See https://github.com/influxdata/telegraf/blob/master/docs/developers/SAMPLE_CONFIG.md#style

Reply:
Just updated it, I think it's good now.